release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2857,6 +2857,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.8
+[id="ocp-4-17-8_{context}"]
+=== RHSA-2024:10818 - {product-title} {product-version}.8 bug fix, and security update advisory
+
+Issued: 11 December 2024
+
+{product-title} release {product-version}.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:10818[RHSA-2024:10818] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:10821[RHBA-2024:10821] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.8 --pullspecs
+----
+
+[id="ocp-4-17-8-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the provider ID for IBMPowerVSMachine's object did not populate properly due to an improper retrieval of the IBM Cloud Workspace ID. As a result, certificate signing requests (CSRs) were pending in the hosted cluster. With this release, the provider ID successfully populates and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44880[*OCPBUGS-44880*])
```

Comment:
```suggestion
* Previously, the provider ID for the `IBMPowerVSCluster` object did not populate properly due to an improper retrieval of the IBM Cloud Workspace ID. As a result, certificate signing requests (CSRs) were pending in the hosted cluster. With this release, the provider ID successfully populates and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44880[*OCPBUGS-44880*])
```
According to the SSG, we should avoid contractions in technical content: https://github.com/openshift/openshift-docs/pull/84251#issuecomment-2536815515. I checked the [PR](https://github.com/openshift/cluster-api-provider-ibmcloud/pull/94) related to this bug, and it mentions the `IBMPowerVSCluster` object. You might want to check with engineering, but I'm wondering if `IBMPowerVSCluster` is correct here instead of `IBMPowerVSMachine`, since I don't find an object with that name.